client/app/vps/veeam/order/vps-order-veeam.controller.js

```diff
@@ -0,0 +1,62 @@
+class VpsOrderVeeamCtrl {
+    constructor ($scope, $stateParams, $translate, $window, CloudMessage, CloudNavigation, VpsService) {
+        "use strict";
+        this.$scope = $scope;
```

Comment:
I am just asking why use $scope in that case. Why not simply use ES6 to do this ?